include/libopencm3/stm32/f0/adc.h

```diff
@@ -80,6 +80,14 @@
 #define ADC_CFGR1_EXTSEL_SHIFT		6
 #define ADC_CFGR1_EXTSEL		(0x7 << ADC_CFGR1_EXTSEL_SHIFT)
 #define ADC_CFGR1_EXTSEL_VAL(x)		((x) << ADC_CFGR1_EXTSEL_SHIFT)
+/** @defgroup adc_cfgr1_extsel ADC external trigger selection values
+ *@{*/
+#define ADC_CFGR1_EXTSEL_TIM1_TRGO	ADC_CFGR1_EXTSEL_VAL(0)
+#define ADC_CFGR1_EXTSEL_TIM1_CC4	ADC_CFGR1_EXTSEL_VAL(1)
+#define ADC_CFGR1_EXTSEL_TIM2_TRGO	ADC_CFGR1_EXTSEL_VAL(2)
+#define ADC_CFGR1_EXTSEL_TIM3_TRGO	ADC_CFGR1_EXTSEL_VAL(3)
+#define ADC_CFGR1_EXTSEL_TIM15_TRGO	ADC_CFGR1_EXTSEL_VAL(4)
+/**@}*/
```

Comment:
I've generally tried to avoid adding this level of detail, as it can baloon out of hand _very_ quickly.  Also, this is _already_ incomplete.  See RM0454 rev5.    This is exactly why I don't like adding these sorts of tables, as they often have variations on families, and density parts in the same family.   RM0454 doesn't even try to describe them in the register description, just refers to a different table:
![image](https://user-images.githubusercontent.com/127907/212305723-7130175c-f882-4579-b900-5f7ff790a095.png)

Reply:
Um....
https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/f1/adc.h#L151
https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/f4/adc.h#L100
https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/f7/adc.h#L97
https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/g4/adc.h#L67
https://github.com/libopencm3/libopencm3/blob/master/include/libopencm3/stm32/l1/adc.h#L129

So you're saying F1/F4/F7/G4/L1 all can have these, but F0/G0/L0 can't. Is that right?